clients/java/src/main/java/io/camunda/client/api/search/enums/AdHocSubprocessActivityResult.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.client.api.search.enums;
+
+public class AdHocSubprocessActivityResult {
+
+  public enum Type {
+    UNSPECIFIED,
```

Comment:
❓Do these enum need the class around them? There

Reply:
I guess it's a matter of taste. You either encode their context with a class or by the name of the enum class if it's standalone 🤷